internal/util/util.go

```diff
@@ -158,6 +157,41 @@ type KernelVersion struct {
 	Backport     bool   // backports have a fixed version/patchlevel/sublevel
 }
 
+// Parses a kernel release version string into:
+// version, patch version, sub version and extra version
+func parseKernelRelease(release string) (int, int, int, int, error) {
```

Comment:
```suggestion
// parseKernelRelease parses a kernel release version string into:
// version, patch version, sub version and extra version.
func parseKernelRelease(release string) (int, int, int, int, error) {
```
just a minor change, golang func doc begins with function name 
Thanks! :slightly_smiling_face: